tests/integration/test_integration.py

```diff
@@ -28,7 +28,7 @@ def test_configure_keeps_api_hostname_and_port():
     assert tracer._writer.agent_url == "http://localhost:{}".format("9126" if AGENT_VERSION == "testagent" else "8126")
     tracer.configure(hostname="127.0.0.1", port=8127)
     assert tracer._writer.agent_url == "http://127.0.0.1:8127"
-    tracer.configure(priority_sampling=True)
+    tracer.configure()
```

Comment:
I think we need to pass in something that will reconfigure the writer to ensure it keeps the same url.

if we don't modify any arguments that would cause a writer recreation, then this test won't validate the expected behavior.

Reply:
Actually passing the priority_sampling argument does not trigger writer creation. This test has been broken for a while. I'll update it to use `appsec_enabled=True`. That should do the trick.